.github/workflows/build.yml

```diff
@@ -26,6 +26,12 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Clean
+        run: dotnet clean -v m
+        
+      - name: Clear cache
+        run: dotnet nuget locals all --clear
+        
```

Comment:
Is it required?

Reply:
Get periodic hit and miss test failures from GitHub actions without it (had one when I was testing) https://github.com/actions/setup-dotnet/issues/155#issuecomment-748452076